README.md

```diff
@@ -151,16 +134,21 @@ For example, the following methods would be mixed into your model class:
 
 For example:
 
-**/server/boot/script.js**
+```ts
+import {juggler} from '@loopback/repository';
+
+const db = new juggler.DataSource({
+  connector: 'rest',
+  debug: false,
+  crud: true,
+  baseURL: 'http://localhost:3000'
+});
+```
 
+// @TODO: Is it useful in LB4?
```

Comment:
Was not able to upgrade from LB3 to LB4 even checking the current code/documentation/issues/....